data/data/bootstrap/files/usr/local/bin/bootstrap-verify-api-server-urls.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC1091
+. /usr/local/bin/bootstrap-service-record.sh
+
```

Comment:
@patrickdillon @jstuever I am unclear about whether kubeconfig needs to be specified here. 
From my understanding, if this script is only run from `analyze` we don't have to but if output from this service is to be included in install-gather, then we would have to?

Reply:
>  I am unclear about whether kubeconfig needs to be specified here.

Every instance of `oc` I see in the bootstrap node specifies a kubeconfig explicitly so I think so.

> if this script is only run from analyze we don't have to but if output from this service is to be included in install-gather, then we would have to?

`analyze` does not execute anything on the bootstrap node. `gather` will execute commands to grab logs on the bootstrap node, but this service should just be running as part of the standard execution of bootkube. `gather` will not cause it to be run but will just collect its logs